src/main/scala/cromwell/engine/db/slick/WorkflowExecutionComponent.scala

```diff
@@ -59,4 +64,34 @@ trait WorkflowExecutionComponent {
       if workflowExecution.status inSet statuses
     } yield workflowExecution
   }
+
+  /**
+   * Query workflow execution using the filter criteria encapsulated by the `WorkflowQueryParameters`.
+   */
+  def queryWorkflowExecutions(queryParameters: WorkflowQueryParameters) = {
+    val Include: Rep[Boolean] = true
```

Comment:
I think of `val`s as immutable variables but not necessarily constants.  In most cases they capture the value of an expression that evaluates differently depending on the values of other variables or (heaven forfend) side effects.  But in this case the `val`s will have the same values every time this program is run.

Having said that I don't really care whether this is capitalized or not.